modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -325,6 +325,20 @@ CV__DNN_INLINE_NS_BEGIN
         static Ptr<PoolingLayerInt8> create(const LayerParams& params);
     };
 
+    class CV_EXPORTS ReduceLayer : public Layer
+    {
+    public:
+        int type;
+        std::vector<size_t> deletedDims;
```

Comment:
> deletedDims

reductionDims? or something else?

Could you please add link to some document, e.g. ONNX specification